.nvmrc

```diff
@@ -1 +1 @@
-v14.17.3
+v16.20.0
```

Comment:
We should also update this one 👇 

https://github.com/oblador/loki/blob/7ca6aa0101151dd7ec56652708cdafd1083079cf/docs/serverless.md?plain=1#L66

Reply:
I'd also suggest adding " (Node.js v16.x or newer)" to the "prerequisites" sections

https://github.com/oblador/loki/blob/7ca6aa0101151dd7ec56652708cdafd1083079cf/packages/loki/README.md?plain=1#L25

https://github.com/oblador/loki/blob/7ca6aa0101151dd7ec56652708cdafd1083079cf/docs/getting-started.md?plain=1#L8